README.md

```diff
@@ -97,7 +97,8 @@ export default AjaxService.extend({
 1.  Create a `gh-pages` branch locally via `git checkout -b gh-pages`.
 1.  **DO NOT PUSH TO GH-PAGES YET**
 1.  Remove `/dist` from `.gitignore` by adding a '#' before it.
-1.  Add and commit this change.
+1.  Add and commit this change. (`git add dist/`)
+<details><summary>*You just said remove, why am I adding again?:*</summary><options> This is because you just removed `dist/` from the gitignore, which means that git is now aware of it and will track it if added. We want to track dist on `gh-pages` so we add and commit it here.*</details>
```

Comment:
I should have known not to trust atom's markdown preview. Making the changes now, thank you!